### ISSUE TITLE
Contribution an Project Plan

### ISSUE THREAD

**Issue by mattmeye** [onury/accesscontrol#42]
please add a way to contribute and a timeline for v3

**Comment by onury** [comment]
See [Contribution](https://github.com/onury/accesscontrol#contributing) section in readme.
Cannot currently provide a timeline since I'm a bit busy these days.

**Comment by scandinave** [comment]
Hi.
What he says, is that it's difficult to help you without a roadmap that describe what is done and what need to be done. 
A distinct branch with your development code for V3 would be great too. 
You have done a really good job, and i think lots of people want/can help you

**Comment by onury** [comment]
Well, when you say "timeline", I think of dates and schedules 😱 
But I can put together a roadmap or todo list, sure.

BTW, I'd really appreciate any kind of help and PRs are always welcome. 
Generally what I do is create a v2, v3 branch like I did with [Docma](https://github.com/onury/docma), then merge it to master. That's what I'll do here once we come to that.

I need to do some organizing first.. I'll update you all on this shortly.
Re-opening this..

**Comment by scandinave** [comment]
Maybe you can use the github project board for this

**Comment by onury** [comment]
Cannot pull/commit/download project section on GitHub yet. That's why I was thinking of a `ROADMAP.md` file (which is also more visible to users if they're willing to contribute). Wouldn't you think so?

**Comment by scandinave** [comment]
Yeah it's also good. Project is a paid function of github?

**Comment by onury** [comment]
No it's not a paid service. But you just cannot manage it in your local repo (just like the issues section).

**Comment by scandinave** [comment]
Ok. Do what is the best for you. It's your project :)

**Comment by onury** [comment]
No. Just indicating my reasoning.. I'm open to suggestions, discussions. I only care to make projects more usable, helpful to people and maintainable. 

If you and more people; who wants to contribute, think some workflow or the other is better, I will sure accept it.

**Comment by rosendi** [comment]
@onury Any updates?

**Comment by benedyktdryl** [comment]
@onury do you still have time to maintain the project? I can see a few small PRs from 2020 which can be merged but they aren't. Would you consider transferring the project to someone else or add some people as maintainers?

**Comment by onury** [comment]
Honestly — gathering scattered notes and half-formed ideas for an open-source project across months, sometimes years, while your main focus is business work, is hard. But v3 has finally shipped (**v3.0.0**), so the "timeline for v3" part is resolved.

Contributions are always welcome: open an issue to talk something through, or send a PR. Closing.

Sorry for the late reply!
